charts/seed-bootstrap/templates/configmap-cluster-idenetity.yaml

```diff
@@ -0,0 +1,8 @@
+kind: ConfigMap
```

Comment:
typo in filename

Reply:
What if such configmap already exists (e.g., in case the seed is a garden)?

Reply:
It will reuse the value from the existing configmap.  https://github.com/gardener/gardener/pull/2471/files#diff-5ca37df1e48dc263572bf7822db58a8aR883